scylla-cql/src/frame/response/error.rs

```diff
@@ -1,9 +1,10 @@
-use crate::errors::{DbError, OperationType, WriteType};
```

Comment:
Typo in `errors: remove scylla-cql::errors module` commit msg:
`coresponds` -> `corresponds`